### ISSUE TITLE
ConformsTo (Partial Conformance)

### ISSUE THREAD

**Issue by clapierre** [schemaorg/schemaorg#3271]
In todays Publishing Community Group A11y Task Force call we were tackling the idea of accessibilityFeature = none and if that would represent that this book doesn’t conform to WCAG-A or AA and the omission of ConformsTo does not mean necessarily that it doesn’t conform to the standard just that the publisher didn’t put anything into that metadata field.

Do you have any objection to adding a new value that a publisher could at least say they are Partially-Conformant-to-WCAG and then they can add those accessibilityFeatures like displayTransformability, etc. and in the accessibilitySummary state why it is not conformant something like "There are no alt text descriptions provided."

**Comment by clapierre** [comment]
Well I guess since we control the values of what goes in conformsTo this is not really a schema.org issue now that I think about it.

we can update the vocabulary and include the option if it makes sense of "Partially-Conformant-to-WCAG" which would be explained in the accessibilitySummary.

Thanks
Charles.
